### DISCUSSION TITLE
How to know if native overlayfs is in use?

### DISCUSSION THREAD

**Discussion by the-sun-will-rise-tomorrow** [containers/podman#22275] [Q&A] [answered]
Hi,

I'm a little confused about the new native overlayfs support.

Specifically, how can someone find if it's being used or not?

As I understand, `podman info --debug` will print `graphDriverName: overlay` in both cases.

Is there a way to know if the FUSE or native version is used?

Also, what is `Native Overlay Diff`? Is that the indicator for whether the native variant is used?

Thanks!

**Comment by Luap99** [comment]
podman info will include the `mount_program = /path/to/fuse-overlayfs` in the output under graphOptions if I remember correctly

**Comment by eriksjolund** [comment, marked as answer]
If you run these commands and get the same result, then native overlayfs is used:
```
$ podman info -f {{.Store.GraphDriverName}}
overlay
$ podman info -f '{{index .Store.GraphStatus "Native Overlay Diff"}}'
true
```

Storage driver   | Result of `podman info -f {{.Store.GraphDriverName}}` | Result of `podman info -f '{{index .Store.GraphStatus "Native Overlay Diff"}}`
----             | ------                                                | -----
native overlayfs | overlay                                               | true
fuse-overlayfs   | overlay                                               | false
VFS              | vfs                                                   | false

For more information, see https://github.com/containers/podman/blob/main/docs/tutorials/performance.md

**Reply by the-sun-will-rise-tomorrow** [reply]
Excellent, thank you!
